index.html

```diff
@@ -11,22 +11,200 @@
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+        </div>
+            </section>
+            <section class="AboutMeHeader">
+                <div class="resume">
+                <h3 class="me">About me</h3>
+                <p class="help">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                   don’t hestiate to contact me.</p>
+                <button class="projects" type="button">Get My Resume</button>
+                </div>
+                <br>
+                <div class="planholder">
+                    <div class="box1">
+                        <div class="skills">
+                            <img src="images/icon-languages.svg" alt="icon languages" class="icon">
+                            <h2 class="skills">Languages</h2>
```

Comment:
- [ ] Please stick to the design as much as possible, you can consider aligning the form properly and adding the background images to the contact section as in the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220609_164051](https://user-images.githubusercontent.com/25227022/172888824-f8c23f13-075c-41f2-9e6e-820ea20a8d58.png)|![Screenshot_20220609_164111](https://user-images.githubusercontent.com/25227022/172888852-a0ad08a0-4ba9-444e-b531-68902abb102a.png)|
|![Screenshot_20220609_164430](https://user-images.githubusercontent.com/25227022/172889067-7a70876c-422b-4166-8fb2-8d8897d11422.png)|![Screenshot_20220609_164516](https://user-images.githubusercontent.com/25227022/172889327-c31649f0-daae-4a45-a30a-f8d79ffaef96.png)|